jina/proto/jina.proto

```diff
@@ -317,6 +317,7 @@ message RequestProto {
             TERMINATE = 0; // shutdown the BasePod
             STATUS = 1; // check the status of the BasePod
             IDLE = 3; // used in ROUTER-DEALER pattern, tells the router that the dealer is idle
+            BUSY = 4; // used in ROUTER-DEALER pattern, tells the router that the dealer is busy (or closed)
```

Comment:
Is it needed to have a Predefined set of commands in our Proto? Can't just the command be a String or an Integer that is interpreted by some known code in some Jina enumerator? Otherwise, we may end up with very large set of Commands, why seems not needed and overkill

Reply:
now we have Request primitive data type that could handle this logic, mapping from string to enumerator to guarantee consistent access to Request type. Like this we can keep a simpler proto definition

Reply:
What I like about enums is that it is more efficient in terms of request size and that it is not allowed to insert any values. Less chance of spelling mistakes.
Once we face the issue of having too many commands, we could rethink this.

Reply:
I think is the good point to rethink this, and embrace Primitive data type.

Reply:
ok, let's do in the future